packages/eslint-plugin-gestalt/src/prefer-heading.test.js

```diff
@@ -0,0 +1,108 @@
+// @flow strict
+import {
+  getPathFormatterByRuleName,
+  getRuleTester,
+  getTestTypePrepender,
+  readTestByPath,
+} from './helpers/testHelpers.js';
+import rule from './prefer-heading.js';
+
+const ruleName = 'prefer-heading';
+const ruleTester = getRuleTester();
+const pathFormatter = getPathFormatterByRuleName(ruleName);
+const validPrepender = getTestTypePrepender('valid');
+const invalidPrependerNoDisallowed = getTestTypePrepender('invalid');
+
+const validCode = readTestByPath(pathFormatter(validPrepender('valid')));
+
+const buildInvalidTestNoDisallowed = (name) =>
+  readTestByPath(pathFormatter(invalidPrependerNoDisallowed(name)));
+
+const invalidNoGestaltImportInput = buildInvalidTestNoDisallowed('no-gestalt-import-input');
+const invalidNoGestaltImportOutput = buildInvalidTestNoDisallowed('no-gestalt-import-output');
+const invalidNoGestaltImportNoA11yLevelOutput = buildInvalidTestNoDisallowed(
```

Comment:
I'm not following here…how do we have test output without corresponding input?

Reply:
DONE